hieradata/development.yaml

```diff
@@ -199,6 +199,8 @@ govuk_lvm::no_op: true
 govuk_mount::no_op: true
 
 govuk_elasticsearch::version: '1.7.5'
+govuk_elasticsearch::cors_enabled: true
+govuk_elasticsearch::cors_allow_headers: 'X-Requested-With, Content-Type, Content-Length, If-Modified-Since'
```

Comment:
I set the defaults to the elasticsearch defaults, but I think I'll change it so it's just a simple feature toggle in the hieradata.